renamed vectorclock functions

aletomsic opened the pull request:
Made vector clock names less confusing.
Solves issue #2

aletomsic committed:
renamed vectorclock functions

peterzeller commented:
In the issue you said, that you followed my renaming suggestion, but here you only renamed the `lt` and `gt` functions. Which one was intended?

aletomsic committed:
renamed vectorclock functions strict_ge to gt and strict_le to lt

bieniusa commented on rebar.config:
```diff
@@ -1,10 +1,10 @@
-]}.
+%%{deps, [
+%%    {lager, "2.0", {git, "git://github.com/basho/lager", {tag, "2.0.3"}}}
+%%]}.
```
Why is this commented?

bieniusa commented on rebar.config:
```diff
@@ -1,10 +1,10 @@
+%%]}.
 
-{erl_opts, [debug_info, warnings_as_errors, {parse_transform, lager_transform}, {i,"../include"}]}.
+%%{erl_opts, [debug_info, warnings_as_errors, {parse_transform, lager_transform}, {i,"../include"}]}.
```
Why is this commented?

aletomsic replied:
Hi, I pushed that change by mistake, but:
1. the current code does not use lager.
2. antidote_utils does not compile (erlang19) if that's not commented out.
3. that's a different version than the one used by antidote.

I think we should remove those lines, do you?
Otherwise I can uncomment them back.

bieniusa commented:
Since this is changing the public interface of the antidote_utils module, does this have an impact Antidote's current main branch?

Sorry, just saw the related pull request on Antidote.

aletomsic commented:
Now I did them all.
For some reason I hadn’t pushed those changes.

> On Oct 21, 2016, at 17:22, Peter Zeller notifications@github.com wrote:
> 
> In the issue you said, that you followed my renaming suggestion, but here you only renamed the lt and gt functions. Which one was intended?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub https://github.com/SyncFree/antidote_utils/pull/6#issuecomment-255406683, or mute the thread https://github.com/notifications/unsubscribe-auth/ACdPQg6EBRCJgkDA_6GPZ32CrXm7Ln0jks5q2NisgaJpZM4KdUOr.

bieniusa committed:
Merge branch 'master' into vectorclock-renaming

# Conflicts:
#	rebar.config

Removed the dependency on lager.

bieniusa commented:
@peterzeller Since you started this renaming issue, are you happy with the current version and the new names?

bieniusa committed:
Simplified Makefile and adapted to rebar3 targets

bieniusa committed:
Removed the antidote_utils.erl file

This seems to be a dummy file, no clue why we would need it.

bieniusa commented:
Does anybody know what this json file is good for? Who is using it?

bieniusa commented:
Can someone give me good reasons why we have this repo?
I am not sure whether the code is really usable outside Antidote (but I might be wrong here).

Anybody fighting hard for keeping this separated from Antidote?

peterzeller commented:
> @peterzeller Since you started this renaming issue, are you happy with the current version and the new names?

I am happy.

> Does anybody know what this json file is good for? Who is using it?

I don't see any Json file in the repository?

> Can someone give me good reasons why we have this repo?

Basically antidote_utils is just a vector-clock library, right? Other apps might also use vector-clocks.

bieniusa committed:
Fixed dialyzer issue

This is just a fix to have dialyzer passing. This approach of
serializing PIDs into JSON seems broken to me.
I added an issue to github.
